### PR TITLE
Missing creatures & rebuild

### PR DESCRIPTION
**Pull request by vomar**

### CODE REVIEW

File: data/monster/Arena/greenhorn/achad.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<monster name="Achad" nameDescription="achad" race="blood" experience="70" speed="185" manacost="0">
+<monster name="Achad" nameDescription="Achad" race="blood" experience="70" speed="185" manacost="0">
```
Comment (raymondtfr) [inline data/monster/Arena/greenhorn/achad.xml]: Strings inside ``nameDescription=""`` are no longer case-sensitive -- all strings are converted to lowercase.

Comment (ranisalt) [inline data/monster/Arena/greenhorn/achad.xml]: @raymondtfr wrong, if there is a `nameDescription` attribute then it is used, else it is `"a " + mType->name;`. Check monsters.cpp L740

Comment (raymondtfr) [inline data/monster/Arena/greenhorn/achad.xml]: Oh, I thought about the name description it shows when you click in a creature, which really is all lower case.

**Comment by DSpeichert** [comment]
@WibbenZ is this something we want/can review? It seems like most of the monsters were updated since then.

**Comment by WibbenZ** [comment]
@DSpeichert IMO it's way to long to review and we probably have most of the changes by now.
Checked Amazon and the intervals are not changed but the grammar is updated

**Comment by DSpeichert** [comment]
@vomar Please resubmit the PR with changes rebased from latest master due to the huge number of conflicts in here.

**Pull request closed without merging**